rules/private/phases/phase_zinc_depscheck.bzl

```diff
@@ -16,10 +16,7 @@ load(
 #
 
 def phase_zinc_depscheck(ctx, g):
-    if _DepsConfiguration not in ctx.attr.scala:
```

Comment:
Similar question here: is the early exit no longer needed because the toolchain will always have the deps configuration?

Reply:
To be honest, I don't understand why this statement was here in the first place. Not only does the toolchain always have this provider, but `ctx.attr.scala` should've as well.